homeassistant/components/ohme/sensor.py

```diff
@@ -0,0 +1,103 @@
+"""Platform for sensor."""
+
+from __future__ import annotations
+
+from collections.abc import Callable
+from dataclasses import dataclass
+from typing import Any
+
+from ohme import ChargerStatus, OhmeApiClient
+
+from homeassistant.components.sensor import (
+    SensorDeviceClass,
+    SensorEntity,
+    SensorEntityDescription,
+    SensorStateClass,
+)
+from homeassistant.const import UnitOfElectricCurrent, UnitOfEnergy, UnitOfPower
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from . import OhmeConfigEntry, OhmeRuntimeData
+from .coordinator import OhmeBaseCoordinator
+from .entity import OhmeEntity
+
+PARALLEL_UPDATES = 0
+
+
+@dataclass(frozen=True, kw_only=True)
+class OhmeSensorDescription(SensorEntityDescription):
+    """Class describing Ohme sensor entities."""
+
+    value_fn: Callable[[OhmeApiClient], Any]
+    is_supported_fn: Callable[[OhmeApiClient], bool] = lambda _: True
+    coordinator_fn: Callable[[OhmeRuntimeData], OhmeBaseCoordinator] = (
+        lambda data: data.charge_session_coordinator
+    )
```

Comment:
If you will add more coordinators, I think it would be more beautiful if we split the lists up and just inject the right one